### ISSUE TITLE
[BUG][SDW] Wrong data after release in sdw_alh_loopback_stream_operations

### ISSUE THREAD

**Issue by jdrzaszcz** [thesofproject/sof#2885]
**Description**
When running simultaneous playback and capture on SdW interface on internal loopback with pause/release on playback stream after release at the beginning comes wrong data.

Example recorded file:
![image](https://user-images.githubusercontent.com/46779093/80371393-22487c80-8892-11ea-8825-69a8b9c58107.png)

**Topology**
```
                pipe_plb
                +------------------------+
+----------+    | +------+   +---------+ |     +-------+      +------+
| Host plb |----->|Buffer|-->| Dai ALH | |---->| PDI A | ---> | DP A |
+----------+    | +------+   +---------+ |     +-------+      +------+
                +------------------------+                       ||
                                                                 ||
                pipe_cap                                         ||
                +------------------------+                       ||
+----------+    | +------+   +---------+ |     +-------+      +------+
| Host cap |<-----|Buffer|<--| Dai ALH | |<----| PDI B |<---- | DP B |
+----------+    | +------+   +---------+ |     +-------+      +------+
                +------------------------+
```

**To Reproduce**
Slim Driver python test from file:
test_07_sdw_alh_loopback_stream_operations.py
e.g. TestSdwAlhLoopbackStreamOperations24000Hz16b16b8ch

**Reproduction Rate**
~30 %

**Expected behavior**
Proper data at the beginning of released stream.

**Environment**
WHL RVP
Commit: de2dcbbc2658c67ed7ab761ea4a52a3c3a3672f8 on SOF master branch

 **Example log for in attachment**
[2020_04_28_07_26_19.zip](https://github.com/thesofproject/sof/files/4543503/2020_04_28_07_26_19.zip)

**Comment by slawblauciak** [comment]
@jdrzaszcz isn't this the same issue you've been noticing when testing the dai groups?

**Comment by plbossart** [comment]
I don't think we clear the PDI fifos for a pause. I think we keep the data as is, so don't push anything out, but the capture stream continues so when the first stream resumes it's not aligned with the capture.

We do clear the PDI when starting playback, it was added in https://github.com/thesofproject/linux/pull/1887/commits/01cc461ba4f57d8dcfdf250d38adce9fb657d189

I think no matter what we do, you should expect a discontinuity on the capture path, unless the sine wave period matches the SSP period (4kHz).

**Comment by jdrzaszcz** [comment]
@slawblauciak No, it's different issue.

**Comment by plbossart** [comment]
@slawblauciak is this issue still relevant and isn't it a 'diagnostic driver' tag that's needed?
